lib/CreateStaff.dart

```diff
@@ -1,16 +1,935 @@
+                              onChanged: (value) => {
+                                setState(() {
+                                  if (EmailValidator.validate(value)) {
+                                    validEmail = true;
+                                  } else {
+                                    validEmail = false;
+                                  }
+                                }),
+                              },
+                            )),
+                        ElevatedButton(
+                          onPressed: () {
+                            if (_formKey.currentState!.validate()) {}
+                            // Validate returns true if the form is valid, or false otherwise.
+                          },
+                          child: Text('Save'),
+                        ),
+                      ]),
+                ),
+                Container(
+                  decoration: BoxDecoration(
+                      color: Colors.black12,
+                      border: Border.all(
+                        color: Colors.black,
+                        width: 1,
+                      )),
+                  child: ExpansionTile(
+                      title: Text(
+                        'Address',
+                        style: TextStyle(
+                            color: Colors.black,
+                            fontSize: 20,
+                            fontWeight: FontWeight.bold),
+                      ),
+                      leading: Icon(
+                        Icons.home,
+                        size: 36.0,
+                      ),
+                      children: <Widget>[
+                        ListTile(
+                          title: Text("Current"),
+                          leading: Radio(
+                            value: 1,
+                            groupValue: val,
+                            onChanged: (value) {
+                              setState(() {
+                                val = 1;
+                              });
+                            },
+                            activeColor: Colors.black,
+                          ),
+                        ),
+                        ListTile(
+                          title: Text("Permanent Address"),
+                          leading: Radio(
+                            value: 2,
+                            groupValue: val,
+                            onChanged: (value) {
+                              setState(() {
+                                val = 2;
+                              });
+                            },
+                            activeColor: Colors.black,
+                          ),
+                        ),
+                        CheckboxListTile(
+                          title: Text("Both address are Same"),
+                          controlAffinity: ListTileControlAffinity.leading,
+                          value: address_Value,
```

Comment:
Add all expansiontiles to a method and use it everywhere

Reply:
ok